<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>180</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTreeView" name="treeView">
            <property name="alternatingRowColors" >
             <bool>true</bool>
            </property>
            <property name="selectionBehavior" >
             <enum>QAbstractItemView::SelectItems</enum>
            </property>
            <property name="horizontalScrollMode" >
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="animated" >
             <bool>false</bool>
            </property>
            <property name="allColumnsShowFocus" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QTableView" name="tblTransform">
                 <size>
                  <width>4</width>
                  <height>4</height>
                 </size>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QPushButton" name="btnAddFrame">
                      <property name="text">
                       <string>Add frame</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="vtk_panel">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>180</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>44</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Actions">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
    <addaction name="insertRowAction"/>
    <addaction name="insertColumnAction"/>
    <addaction name="separator"/>
    <addaction name="removeRowAction"/>
    <addaction name="removeColumnAction"/>
    <addaction name="separator"/>
    <addaction name="insertChildAction" />    
   </widget>
   <addaction name="menu_Actions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="insertRowAction">
   <property name="text">
    <string>Insert row</string>
   </property>
  </action>
  <action name="insertColumnAction">
   <property name="text">
    <string>Insert column</string>
   </property>
  </action>
  <action name="removeRowAction">
   <property name="text">
    <string>Remove row</string>
   </property>
  </action>
  <action name="removeColumnAction">
   <property name="text">
    <string>Remove column</string>
   </property>
  </action>
  <action name="insertChildAction" >
   <property name="text" >
    <string>Insert Child</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
